uninstall.ab

```diff
@@ -0,0 +1,12 @@
+
+let amber_path = $command -v amber$ failed {
```

Comment:
Please:
1. Don't leave blank lines at top of file.
2. Consider adding a shebang line and making the file executable.
3. Consider moving Amber code to a `main` block.
4. Add inner spaces in commands `$ ... $`.
```
#!/usr/bin/env amber

main {
    let amber_path = $ command -v amber $ failed {
        echo "Amber is not in $PATH."
        exit 1
    }

    let amber_dir = trust $ dirname "\$(readlink -f {amber_path})" $

    echo "Uninstalling Amber..."
    trust $ rm -f {amber_path} $
    trust $ rm -rf {amber_dir} $
    echo "Amber uninstalled. See you later, 🐊"
}
```